### ISSUE TITLE
ObjectBox crash - java.lang.IllegalStateException ToOne object is null

### ISSUE THREAD

**Issue by shimiuh** [objectbox/objectbox-java#307]
## Issue Basics
- ObjectBox version  ext.objectboxVersion = '1.2.0-RC'
- Reproducibility: not. it is in the playstore getting crashess

### Description
ObjectBox crash - java.lang.IllegalStateException ToOne object is null

### Code
I've got 2 objects with the parent holding a list of children:

```
@Entity
public class Parent {

    @Backlink
    public ToMany<Child> children;
    ...
}
```
and

```
@Entity
public class Child {

    ToOne<Parent> parent;
    ...
}
```

### Logs & stackstraces
```
Caused by java.lang.IllegalStateException
ToOne object is null inside ...Child
io.objectbox.Cursor.nativeGetBacklinkEntities (Cursor.java)
io.objectbox.Cursor.getBacklinkEntities (Cursor.java:279)
io.objectbox.Box.internalGetBacklinkEntities (Box.java:623)
io.objectbox.relation.ToMany.ensureEntities (ToMany.java:172)
io.objectbox.relation.ToMany.size (ToMany.java:395)
```

## Entities
Then I call size() on the ToMany
parent.children.size()

## Misc
It looks like the parent is looking at the list of children and for each one checks the link to the parent. It crashes because this link is null. I don't understand how ObjectBox would manage to get to such a situation. Any ideas?

**Comment by greenrobot-team** [comment]
The same issue was reported in #255 and was reported as fixed as of version `1.2.1`. Please try the [latest version of ObjectBox](https://github.com/objectbox/objectbox-java#objectbox-java-kotlin-android) to see if this issue was resolved. -ut

**Comment by greenrobot-team** [comment]
Closing. Feel free to re-open if you have this issue with `1.4.1` or later. -ut
